### PR TITLE
Feature/landing page (for visitor)

### LINKED ISSUE #31: Create home/landing page
As a user, I would like to be able to see an overview of TripTime features on its homepage, so that I know why should I register an account and use TripTime services.

**Acceptance Criteria:**
- Page should look similar to drawn mockup

**Notes:**

### PR DESCRIPTION
**Pull request by niuniuanran**
Visitor's landing page created

closes #31

### CODE REVIEW

File: src/spa/components/landing/SiteInfo.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <p>
+              <a href='https://github.com/tantigers/TripTime'>Code on Github</a>
+            </p>
+          </div>
+
+          <div className={infoStyles.follow}>
+            <h2>Follow Us</h2>
+            <div className={infoStyles.icons}>
+              <a href='https://github.com/tantigers'>
+                <div>
```
Comment (rafiazman) [inline src/spa/components/landing/SiteInfo.js:47]: Are these to get rid of the underline/text-decoration of <a> tags?

File: src/spa/components/landing/LandingSlide.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+const previewStyles = {
+  'TripTime!': styles.logoPreview,
+  'Review Timeline': styles.timelinePreview,
+  'Plot Destinations': styles.mapPreview,
+  Collaborate: styles.chatPreview,
+};
+
+class LandingSlide extends React.Component {
+  render() {
+    const intro = this.props.intro;
```
Comment (rafiazman) [inline src/spa/components/landing/LandingSlide.js:18]: I would add a property (perhaps boolean?) to determine if the image should be rendered on the left or right side of the screen for desktops

File: src/spa/components/landing/SiteInfo.js (hunk 1 above)
Comment (niuniuanran) [reply src/spa/components/landing/SiteInfo.js:47]: They are redundant now without text captions, removed on commit edff9f76d6fd2dd080233d96fd57ca5bc4a9f2e4

File: src/spa/components/landing/LandingSlide.js (hunk 2 above)
Comment (niuniuanran) [reply src/spa/components/landing/LandingSlide.js:18]: done

**Pull request merged**